app/config/parameters.yml.dist

```diff
@@ -8,6 +8,6 @@ parameters:
     database_driver: pdo_mysql
     database_host: localhost
     database_port: ~
-    database_name: ezstudio
+    database_name: ezplatform-ee
```

Comment:
Not 100% sure, but I think the database name should not contain a hyphen (see https://bugs.mysql.com/bug.php?id=461), probably better to replace with `ezplatform_ee` (also in the other PR).